docs/tutorial_basics.rst

```diff
@@ -27,32 +27,33 @@ We will create a very simple bot that checks our current mood and tries to
 cheer us up if we are feeling sad. It will query our mood and based on our
 reply will respond with a funny image or a message.
 
-Let's start by creating a project folder:
+We created a `starter pack <https://github.com/RasaHQ/starter-pack>`_ to help
```

Comment:
So now we have that code twice? Once in this repo and once in the starter pack?

Reply:
what do you mean? the starter pack is just the bare setup of files, the actual moodbot code is not present in there

Reply:
ah ok, so you still need to create the data files? sounds good :+1: - was just worried we keep that data in two places.

Reply:
no no, all good :D this is the starter pack from the live demo haha